deploy/k8s/sensor.sh

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env bash
+# shellcheck disable=SC1091
+# shellcheck disable=SC2086
```

Comment:
Where do we need this?

Reply:
Its needed for ```ROX_ADMIN_PASSWORD="$(cat ${K8S_DIR}/central-deploy/password)"```. I will move the suppress comment to appropriate position.